resources/qml/TimelineView.qml

```diff
@@ -298,6 +301,58 @@ Item {
         onClicked: Rooms.resetCurrentRoom()
     }
 
+    ParticleSystem { id: confettiParticleSystem }
+
+    Emitter {
+        id: confettiEmitter
+
+        width: parent.width * 3/4
+        enabled: false
+        anchors.horizontalCenter: parent.horizontalCenter
+        y: parent.height
+        emitRate: Math.min(400 * Math.sqrt(parent.width * parent.height) / 870, 1000)
+        lifeSpan: 15000
+        system: confettiParticleSystem
+        velocityFromMovement: 8
+        size: 16
+        sizeVariation: 4
+        velocity: PointDirection {
+            x: 0
+            y: -Math.min(450 * parent.height / 700, 1000)
+            xVariation: Math.min(4 * parent.width / 7, 450)
+            yVariation: 250
+        }
+
+        ImageParticle {
```

Comment:
Why are you using an image for a rectangle? Because of the colors?

Reply:
Yes

Reply:
It's been a while since I wrote that code but I recall the image particle being the best/only way to get what I wanted.